services/credentials-service/package.json

```diff
@@ -35,6 +35,7 @@
     "@nestjs/swagger": "^6.1.4",
     "@nestjs/terminus": "^9.1.4",
     "@prisma/client": "4.8.1",
+    "@techsavvyash/bitstring": "^3.2.0",
```

Comment:
What does this package do? Is there anything more standard that does the same thing?

Reply:
This package is an implementation of the bitstring data structure for the revocation list implementation. This is a direct fork of https://github.com/digitalbazaar/bitstring